tests/semantic_analyser.cpp

```diff
@@ -590,6 +591,14 @@ TEST(semantic_analyser, call_zero)
   test("kprobe:f { @x = count(); zero(@x) ? 0 : 1; }", 10);
 }
 
+TEST(semantic_analyser, call_len)
+{
+  test("kprobe:f { @x[0] = 0; len(@x); }", 0);
+  test("kprobe:f { @x[0] = 0; len(); }", 1);
+  test("kprobe:f { @x[0] = 0; len(@x, 1); }", 1);
+  test("kprobe:f { $x = 0; len($x); }", 1);
```

Comment:
Add negative test for `len(@x[2])` ? I think I saw handling for map args in semantic analyser

Reply:
Added